provision/install-java.sh

```diff
@@ -1,26 +1,13 @@
 #!/bin/sh
 
 set -x
```

Comment:
could you set this to `set -ex` here as well - when I was poking at this yesterday that lead to silent failures

Reply:
https://github.com/vimc/montagu-ci/pull/23/commits/b5482dbe53b6ff950c1ee8317c1b6dacc8c33a7a